app/models/template_section.rb

```diff
@@ -5,7 +5,6 @@ module FlexCommerce
   #
   # This model provides access to the flex commerce template section
   #
-  #
   class TemplateSection < FlexCommerceApi::ApiBase
     belongs_to :template_definition, class_name: "::FlexCommerce::TemplateDefinition"
```

Comment:
This is what the PR is for `belongs_to :template_definition, class_name: "::FlexCommerce::TemplateDefinition"` but the change is already in the feature branch that I'm merging into.